airflow/hooks/jdbc_hook.py

```diff
@@ -57,3 +57,16 @@ def set_autocommit(self, conn, autocommit):
         :return:
         """
         conn.jconn.setAutoCommit(autocommit)
+
+    def get_autocommit(self, conn):
+        """
+        Get autocommit setting for the provided connection.
+        Return True if conn.autocommit is set to True.
+        Return False if conn.autocommit is not set or set to False
+
+        :param conn: The connection
```

Comment:
Could you also add the type, please? Either via the docs `:type` or in the function header directly to the arg `conn:`